config/editor_profiles/default/configurations/SourceLabels.yml

```diff
@@ -3980,15 +3971,6 @@ smn:
     pt: Inari Sami
     es: Inari Sami
     pl: Samski inarski
-srp:
-  label:
-    IT: Serbo
-    FR: Serbe
-    de: Serbisch
-    en: Serbian
-    pt: "Sérvio"
-    es: Serbio
-    pl: Serbski
```

Comment:
Is this one a duplicate?

Reply:
Yes:

https://github.com/rism-ch/muscat/blob/develop/config/editor_profiles/default/configurations/SourceLabels.yml#L3983

https://github.com/rism-ch/muscat/blob/develop/config/editor_profiles/default/configurations/SourceLabels.yml#L4019